docs/source/notes/distributed_training.rst

```diff
@@ -88,6 +88,17 @@ accumulation, and vice versa. Our trainer strives to respect this equivalency
 and ensure identical behavior regardless of the combinations of space and time
 parallelization used.
 
+Distributed Sampling
+--------------------
+
+When providing :class:`torch.utils.data.IterableDataset` with :class:`torch.utils.data.DataLoader`
```

Comment:
Is this supposed to be `torch.utils.data.Dataset`?

Reply:
Only IterableDataset requires DistributedSampler iirc

Reply:
Nvm I had it backwards